corehq/messaging/smsbackends/vertex/models.py

```diff
@@ -0,0 +1,32 @@
+from corehq.messaging.smsbackends.http.models import SQLSMSBackend
+from corehq.messaging.smsbackends.vertex.forms import VertexBackendForm
+
+
+class SQLVertexBackend(SQLSMSBackend):
```

Comment:
we can remove the SQL from the beginning of SQLVertexBackend - it's on other classes because I haven't had a chance to rename them from when I moved everything over from couch